src/test/resources/answer/insert_1_t_alter.err_answer

```diff
@@ -1 +1 @@
-Error: Error while compiling statement: FAILED: HiveAccessControlException Permission denied: user [spark] does not have [UPDATE] privilege on [spark_ranger_test/t_alter] (state=42000,code=40000)
\ No newline at end of file
+Error: shadehive.org.apache.hive.service.cli.HiveSQLException: Error while compiling statement: FAILED: HiveAccessControlException Permission denied: user [spark] does not have [UPDATE] privilege on [spark_ranger_test/t_alter] (state=,code=0)
```

Comment:
Oh, is this missing part of the previous DBCP?
- "shadehive.org.apache.hive.service.cli.HiveSQLException"

Reply:
Please exclude all files include this `shadehive.org.apache.hive.service.cli.HiveSQLException`.